src/components/BuildConfigBuildMetricsPage/BuildConfigBuildMetricsPage.tsx

```diff
@@ -0,0 +1,37 @@
+import { useEffect } from 'react';
+
+import { BUILD_METRICS_PAGE_REQUEST_CONFIG, PageTitles } from 'common/constants';
+
+import { useParamsRequired } from 'hooks/useParamsRequired';
+import { useServiceContainer } from 'hooks/useServiceContainer';
+
+import { BuildMetrics } from 'components/BuildMetrics/BuildMetrics';
+import { ContentBox } from 'components/ContentBox/ContentBox';
+import { ServiceContainerLoading } from 'components/ServiceContainers/ServiceContainerLoading';
+
+import * as buildConfigApi from 'services/buildConfigApi';
+
+export const BuildConfigBuildMetricsPage = () => {
+  const { buildConfigId } = useParamsRequired();
+
+  const serviceContainerBuilds = useServiceContainer(buildConfigApi.getBuilds);
+  const serviceContainerBuildsRunner = serviceContainerBuilds.run;
+
+  useEffect(() => {
+    serviceContainerBuildsRunner({ serviceData: { id: buildConfigId }, requestConfig: BUILD_METRICS_PAGE_REQUEST_CONFIG });
+  }, [serviceContainerBuildsRunner, buildConfigId]);
+
+  return (
+    <>
+      <ServiceContainerLoading {...serviceContainerBuilds} title={PageTitles.buildMetrics}>
+        {serviceContainerBuilds?.data?.content && (
```

Comment:
`serviceContainerBuilds?.data?.content` check should not be used, because the conditionally rendered component is inside `ServiceContainerLoading` children

Reply:
this was not addressed.

Reply:
there is a `TypeError: Cannot read properties of undefined (reading 'content')` when removing type check with ``` <BuildMetrics builds={serviceContainerBuilds!.data!.content!} chartType="line" componentId="bm-l" />```

Reply:
Okay, I did not realize content could be empty.